modeltranslation/static/modeltranslation/js/tabbed_translation_fields.js

```diff
@@ -212,9 +240,9 @@ var google, django, gettext;
           container.find("script").remove();
           panel = $('<div id="' + tabId + '"></div>').append(container);
           tab = $(
-            "<li" +
-              (label.hasClass("required") ? ' class="required"' : "") +
-              '><a href="#' +
+            `<li class='${selectors["tabLiClass"]} ` +
+              (label.hasClass("required") ? "required" : "") + "'" +
+              `><a class="${selectors["tabAClass"]}" href="#` +
```

Comment:
I think it looks good, except that now we have string templates mixed with concatenation.

But that's for another refactoring.